src/routes/Receive.tsx

```diff
@@ -365,43 +347,38 @@ export default function Receive() {
                         </Match>
                         <Match when={unified() && receiveState() === "show"}>
                             <FeeWarning fee={lspFee()} flavor={flavor()} />
-                            <IntegratedQr
```

Comment:
Not sure how this happened but you seem to be reverting some work here. (the new QR design)